torchci/lib/drciUtils.ts

```diff
@@ -24,14 +29,73 @@ Note: Links to docs will display an error until the docs builds have been comple
 }
 
 export function formDrciComment(
-    pr_num: number,
-    pr_results: string = "",
+  pr_num: number,
+  pr_results: string = "",
+  sevs: string = ""
 ): string {
-    const header = formDrciHeader(pr_num);
-    const comment =
-        `${DRCI_COMMENT_START}
+  const header = formDrciHeader(pr_num);
+  const comment = `${DRCI_COMMENT_START}
 ${header}
+${sevs}
 ${pr_results}
 ${DRCI_COMMENT_END}`;
-    return comment;
+  return comment;
+}
+
+
+export async function getDrciComment(
+  octokit: Octokit,
+  owner: string,
+  repo: string,
+  prNum: number
+): Promise<{ id: number; body: string }> {
+  const commentsRes = await octokit.rest.issues.listComments({
+    owner: owner,
+    repo: repo,
+    issue_number: prNum,
+  });
+  for (const comment of commentsRes.data) {
+    if (comment.body!.includes(DRCI_COMMENT_START)) {
+      return { id: comment.id, body: comment.body! };
+    }
+  }
+  return { id: 0, body: "" };
+}
+
+export function getActiveSEVs(issues: IssueData[]): string {
+  const activeSEVs = issues.filter(
+    (issue: IssueData) => issue.state === "open"
+  );
+  if (activeSEVs.length === 0) {
+    return "";
+  }
+  const [merge_blocking, not_merge_blocking] = _.partition(
+    activeSEVs,
+    (issue: IssueData) => issue.body.toLowerCase().includes("merge blocking")
+  );
+  const sev_list = merge_blocking
+    .concat(not_merge_blocking)
+    .map(
+      (issue: IssueData) =>
+        `* ${
+          issue.body.toLowerCase().includes("merge blocking")
+            ? "(merge blocking) "
+            : ""
+        }[${issue.title}](${issue.html_url.replace(
+          "github.com",
+          "hud.pytorch.org"
```

Comment:
I notice that you replace the url here and create a redirection page back to GitHub below.  Are there any particular reasons for that?  The guess I have in mind is that you want to track how many people click on the link in the future?

Reply:
if I link the github issue directly, the PR would get mentioned in the sev issue, and even editing the comment to get rid of the link won't get rid of the mention, and so the sev issue would be covered in a ton of unrelated PRs

Reply:
Ohhh, that was why you mention this earlier in the chat. Got it!